### PR TITLE
Add more requests and reorganize network tab

### PR DESCRIPTION
**Pull request by TheGreenBandit**

### CODE REVIEW

File: src/backend/looped/self/off_radar.cpp [hunk 1]
```diff
@@ -12,13 +12,16 @@ namespace big
...
+			MISC::SET_BIT(script_global(2794162).at(4667).as<int*>(), 2);
+			*scr_globals::globalplayer_bd.at(PLAYER::PLAYER_ID(), 463).at(210).as<bool*>() = true;
+			*script_global(2672505).at(57).as<int*>() = NETWORK::GET_NETWORK_TIME() + 1;
 		}
 
 		virtual void on_disable() override
 		{
-			scr_globals::globalplayer_bd.as<GlobalPlayerBD*>()->Entries[self::id].OffRadarActive = false;
+			MISC::CLEAR_BIT(script_global(2794162).at(4667).as<int*>(), 2);
+			*scr_globals::globalplayer_bd.at(PLAYER::PLAYER_ID(), 463).at(210).as<bool*>() = false;
```
Comment (Yimura) [inline src/backend/looped/self/off_radar.cpp]: What does this change do? If this is Ghost Organisation then add a toggle for it.

File: src/views/network/view_network.cpp [hunk 2]
```diff
@@ -441,53 +473,21 @@ namespace big
...
 		ImGui::Checkbox("DISABLE_CEO_MONEY"_T.data(), &g.session.block_ceo_money);
 		if (ImGui::IsItemHovered())
 			ImGui::SetTooltip("DISABLE_CEO_MONEY_DESC"_T.data());
 		ImGui::SameLine();
-		ImGui::Checkbox("RANDOMIZE_CEO_COLORS"_T.data(), &g.session.randomize_ceo_colors);
 		ImGui::Checkbox("Block Jobs", &g.session.block_jobs);
+
+		ImGui::Checkbox("Random CEO Colors", &g.session.randomize_ceo_colors);
 		if (ImGui::IsItemHovered())
 			ImGui::SetTooltip("Prevents remote players from starting jobs while in your session");
```
Comment (Yimura) [inline src/views/network/view_network.cpp]: What does Random CEO colors have to do with the description set by IsItemHovered?

**Review by Yimura — CHANGES_REQUESTED** [review]

File: src/views/network/view_network.cpp [hunk 3]
```diff
@@ -441,53 +473,21 @@ namespace big
...
-
 		components::sub_title("SCRIPT_HOST_FEATURES"_T);
 		ImGui::Checkbox("DISABLE_CEO_MONEY"_T.data(), &g.session.block_ceo_money);
 		if (ImGui::IsItemHovered())
 			ImGui::SetTooltip("DISABLE_CEO_MONEY_DESC"_T.data());
 		ImGui::SameLine();
-		ImGui::Checkbox("RANDOMIZE_CEO_COLORS"_T.data(), &g.session.randomize_ceo_colors);
 		ImGui::Checkbox("Block Jobs", &g.session.block_jobs);
+
+		ImGui::Checkbox("Random CEO Colors", &g.session.randomize_ceo_colors);
```
Comment (tupoy-ya) [inline src/views/network/view_network.cpp]: Why did you remove translation here?

**Comment by Yimura** [comment]
Pictures could also be helpful to show what you've done.

File: src/util/mobile.hpp [hunk 4]
```diff
@@ -101,6 +101,36 @@ namespace big::mobile
...
+
+		inline void request_acidlab()
+		{
+			*script_global(scr_globals::mechanic_global).at(944).as<int*>() = 1;
+		}
+
+		inline void request_acidlab_bike()
+		{
+			*script_global(scr_globals::mechanic_global).at(994).as<int*>() = 1;
+		}
```
Comment (tupoy-ya) [inline src/util/mobile.hpp:134]: I don't think those are `ceo_abilities`, you can call them while not being CEO.

File: src/views/network/view_network.cpp [hunk 5]
```diff
@@ -22,7 +22,7 @@ namespace big
 	{
 		ImGui::BeginGroup();
 		components::sub_title("Rid joiner");
-		if (ImGui::BeginListBox("##ridjoiner", get_listbox_dimensions()))
+		if (ImGui::BeginListBox("##ridjoiner", {.5f * 750, 195.f}))
```
Comment (tupoy-ya) [inline src/views/network/view_network.cpp]: Why do that?

**Review by tupoy-ya — CHANGES_REQUESTED** [review]

File: src/views/network/view_network.cpp (hunk 3 above)
Comment (TheGreenBandit) [reply src/views/network/view_network.cpp:483]: will add back, was testing sizes

File: src/views/network/view_network.cpp (hunk 5 above)
Comment (TheGreenBandit) [reply src/views/network/view_network.cpp:25]: testing sizes again lol

**Comment by TheGreenBandit** [comment]
> Pictures could also be helpful to show what you've done.

Your right, forgot about that. https://imgur.com/gallery/XqQHXbZ

**Comment by Yimura** [comment]
> Your right, forgot about that. https://imgur.com/gallery/XqQHXbZ

404 error

**Comment by TheGreenBandit** [comment]
was removed by imgur staff. Will upload a different way

File: src/backend/looped/self/off_radar.cpp [hunk 6]
```diff
@@ -12,15 +13,20 @@ namespace big
...
 		{
-			scr_globals::globalplayer_bd.as<GlobalPlayerBD*>()->Entries[self::id].OffRadarActive = false;
+			if (!g.self.ghost_org)
+				MISC::CLEAR_BIT(script_global(2794162).at(4667).as<int*>(), 2);
+			*scr_globals::globalplayer_bd.at(PLAYER::PLAYER_ID(), 463).at(210).as<bool*>() = false;
 		}
 	};
 
 	off_radar g_off_radar("otr", "OFF_RADAR", "OFF_RADAR_DESC", g.self.off_radar);
+	bool_command ghost_org("ghostorg", "Ghost Org", "Ghost Organization for free.", g.self.ghost_org);
```
Comment (Yimura) [inline src/backend/looped/self/off_radar.cpp]: No need to add "for free", perhaps write "Use Ghost Organisation instead of standard off radar."

I'm open for other suggestions.

**Review by Yimura — CHANGES_REQUESTED** [review]

File: src/backend/looped/self/off_radar.cpp (hunk 6 above)
Comment (TheGreenBandit) [reply src/backend/looped/self/off_radar.cpp:31]: I also realized i forgot to add it to the self view lmao

File: src/backend/looped/self/off_radar.cpp [hunk 7]
```diff
@@ -12,15 +13,20 @@ namespace big
 
 		virtual void on_tick() override
 		{
-			scr_globals::globalplayer_bd.as<GlobalPlayerBD*>()->Entries[self::id].OffRadarActive = true;
-			*scr_globals::offradar_time.at(57).as<int*>() = NETWORK::GET_NETWORK_TIME() + 1;
+			if (g.self.ghost_org)
+				MISC::SET_BIT(script_global(2794162).at(4667).as<int*>(), 2);
+			*scr_globals::globalplayer_bd.at(PLAYER::PLAYER_ID(), 463).at(210).as<bool*>() = true;
```
Comment (maybegreat48) [inline src/backend/looped/self/off_radar.cpp]: Should still be 
```c++
scr_globals::globalplayer_bd.as<GlobalPlayerBD*>()->Entries[self::id].OffRadarActive = true;
*scr_globals::offradar_time.at(57).as<int*>() = NETWORK::GET_NETWORK_TIME() + 1;
```

File: src/backend/looped/self/off_radar.cpp [hunk 8]
```diff
@@ -12,15 +13,20 @@ namespace big
...
+			*script_global(2672505).at(57).as<int*>() = NETWORK::GET_NETWORK_TIME() + 1;
 		}
 
 		virtual void on_disable() override
 		{
-			scr_globals::globalplayer_bd.as<GlobalPlayerBD*>()->Entries[self::id].OffRadarActive = false;
+			if (!g.self.ghost_org)
+				MISC::CLEAR_BIT(script_global(2794162).at(4667).as<int*>(), 2);
+			*scr_globals::globalplayer_bd.at(PLAYER::PLAYER_ID(), 463).at(210).as<bool*>() = false;
```
Comment (maybegreat48) [inline src/backend/looped/self/off_radar.cpp]: Should still be
```c++
scr_globals::globalplayer_bd.as<GlobalPlayerBD*>()->Entries[self::id].OffRadarActive = false;
```

File: src/util/mobile.hpp [hunk 9]
```diff
@@ -101,6 +101,36 @@ namespace big::mobile
...
+		{
+			*script_global(scr_globals::mechanic_global).at(960).as<int*>() = 1;
+		}
+
+		inline void request_mobile_operations_center()
+		{
+			*script_global(scr_globals::mechanic_global).at(930).as<int*>() = 1;
+		}
+
+		inline void request_terrobyte()
```
Comment (maybegreat48) [inline src/util/mobile.hpp]: *terrorbyte

File: src/util/mobile.hpp [hunk 10]
```diff
@@ -101,6 +101,36 @@ namespace big::mobile
 		{
 			*script_global(scr_globals::mechanic_global).at(896).as<int*>() = 1;
 		}
+
```
Comment (maybegreat48) [inline src/util/mobile.hpp:103]: These should be commands

File: src/backend/looped/self/off_radar.cpp (hunk 7 above)
Comment (TheGreenBandit) [reply src/backend/looped/self/off_radar.cpp:18]: forgot i was testing something

**Comment by gir489returns** [comment]
> was removed by imgur staff. Will upload a different way
> 
> EDIT -> https://cdn.discordapp.com/attachments/1010434493997387837/1129349435256750250/networkupdate.png

https://github.com/massgravel/Microsoft-Activation-Scripts/

Activate windows already.

**Comment by TheGreenBandit** [comment]
> > was removed by imgur staff. Will upload a different way
> > EDIT -> https://cdn.discordapp.com/attachments/1010434493997387837/1129349435256750250/networkupdate.png
> 
> https://github.com/massgravel/Microsoft-Activation-Scripts/
> 
> Activate windows already.

unactivated for half a year and going 💪

**Review by Yimura — CHANGES_REQUESTED** [review]
Changes from maybegreat haven't been implemented.

For example the off radar one:
```cpp
scr_globals::globalplayer_bd.as<GlobalPlayerBD*>()->Entries[self::id].OffRadarActive = false;
```

**Review by Yimura — APPROVED** [review]
